notebooks/abbash_notebook.ipynb

```diff
@@ -2,21 +2,118 @@
  "cells": [
```

Comment:
<p>Its best to write the tutorial in the 2nd person, i.e.</p><ul><li>"you will take a look"</li><li>"You will define a sine wave" or "Define a sine wave"</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>